comms/src/connection/peer_connection/worker.rs

```diff
@@ -238,7 +238,7 @@ impl PeerConnectionWorker {
 
     /// Handles socket events from the ConnectionMonitor. Updating connection
     /// state as necessary.
-    fn handle_socket_event(&mut self, event: SocketEvent) -> Result<()> {
+    fn handle_socket_event(&mut self, event: SocketEvent) -> Result<bool> {
```

Comment:
You could update the docstring with the meaning of the bool return flag

Reply:
Ah thought I removed that - good spot